src/shared/modules/guides/guidesDuck.ts

```diff
@@ -22,61 +22,120 @@ import docs from 'browser/documentation'
 import { GlobalState } from 'shared/globalState'
 
 export const NAME = 'guides'
-export const START_GUIDE = 'sidebar/START_GUIDE'
+export const SET_GUIDE = 'sidebar/SET_GUIDE'
 export const GOTO_SLIDE = 'sidebar/GOTO_SLIDE'
+export const UPDATE_REMOTE_GUIDES = 'sidebar/UPDATE_REMOTE_GUIDES'
+export const ADD_GUIDE = 'sidebar/ADD_GUIDE'
 
-export const isDefaultGuide = (guide: Guide): boolean =>
-  guide.title === defaultGuide.title
+export const getCurrentGuide = (state: GlobalState): Guide | null =>
+  state[NAME].currentGuide
+export const getRemoteGuides = (state: GlobalState): Guide[] =>
+  state[NAME].remoteGuides
 
-export const getGuide = (state: GlobalState): Guide => state[NAME].guide
 export type Guide = {
   currentSlide: number
   title: string
   slides: JSX.Element[]
 }
 
-const defaultGuide: Guide = {
-  ...docs.guide.chapters.index,
-  currentSlide: 0
-}
 export interface GuideState {
-  guide: Guide
+  currentGuide: Guide | null
+  remoteGuides: Guide[]
```

Comment:
We'll need to make sure this is saved/loaded to/from localStorage, and that it doesn't break when it is missing in localstorage (it seems guides were not persisted in previous browser versions)